tests/safeds/data/tabular/containers/_column/test_transform.py

```diff
@@ -0,0 +1,9 @@
+from safeds.data.tabular.containers import Column
+
+
+def test_should_transform_column() -> None:
+    column = Column("test", [1, 2])
+    column = column.transform(lambda it: it + 1)
+
+    assert column[0] == 2
```

Comment:
Instead of comparing values element-wise, create an expected Column and compare with that. This would have also caught the issue that you return a `Series` instead of a `Column`.